templates/store.html

```diff
@@ -6,13 +6,17 @@ <h1>Algorithms store</h1>
 <input type="button" onclick="location.href='/add_alg'" value="Add new algorithm" />
 <br><br>
 
-{% for algorithm in algorithms_list %}
+{% for algorithm in algorithms_preview_list %}
     <div class='algorithm_shortview'>
         <h2>{{ algorithm.title }}</h2>
         <p>{{ algorithm.description }}</p>
         <p>cost: {{ algorithm.cost }}</p>
         <!--to do: check for unavailable algorithms-->
```

Comment:
remove TODO since it's implemented in this PR.

Reply:
Done.